pkg/hcl_interpreter/moduletree.go

```diff
@@ -184,18 +184,9 @@ func (mtree *ModuleTree) Errors() []error {
 	missingModules := mtree.meta.MissingRemoteModules
 	if len(missingModules) > 0 {
 		missingModulesList := strings.Join(missingModules, ", ")
-		firstSentence := "Could not load some remote submodules"
-		if mtree.meta.Dir != "." {
-			firstSentence += fmt.Sprintf(
-				" that are used by '%s'",
-				mtree.meta.Dir,
-			)
-		}
 
 		errors = append(errors, fmt.Errorf(
-			"%s. Run 'terraform init' if you would like to include them in the evaluation: %s",
-			firstSentence,
-			missingModulesList,
+			"%w: %s: %s", ErrLoadRemoteSubmodules, mtree.meta.Dir, missingModulesList,
```

Comment:
@jason-snyk would I be able to access the `mtree.meta.Dir` and `missingModulesList` values when receiving this error?
I would like to add in `snyk-iac-test` a logic that is very similar to what you did here before my change.

Reply:
I don't think those properties make it out anywhere that you'd be able to access outside of this package. What I would do in this case is create a custom error struct with those properties, [like in this tutorial](https://dev.to/tigorlazuardi/go-creating-custom-error-wrapper-and-do-proper-error-equality-check-11k7). Then you can use `errors.As` in `snyk-iac-test` to cast the `error` back into your custom struct and access those properties.

Reply:
I ended up transitioning all of the user-facing errors to a custom error struct.